frontend/src/types/Action.ts

```diff
@@ -0,0 +1,26 @@
+import { ExplainerProps } from "@/components/Explainer/Explainer";
+import { FinalProps } from "@/components/Final/Final";
+import { InfoProps } from "@/components/Info/Info";
+import { LoadingProps } from "@/components/Loading/Loading";
+import { PlaylistProps } from "@/components/Playlist/Playlist";
+import { RedirectProps } from "@/components/Redirect/Redirect";
+import { ScoreProps } from "@/components/Score/Score";
+import { TrialProps } from "@/components/Trial/Trial";
```

Comment:
Yeah, so here I'm importing the different action type props from their components, but I'm thinking about importing them into here as they are ultimately props/data that come directly from the backend.

Reply:
I agree that it would be clearer to have all different types of `Action` in one place.